libraries/saddiamondman's unorganised folder/assets/sprites/tutorial_masters/bepis_master/hat_5.png

```diff
@@ -0,0 +1 @@
+hat_3.png
```

Comment:
The symlinks do not resolve in the zip download
![image](https://github.com/user-attachments/assets/825edb99-3e04-499a-809a-6141c094ba11)